src/main/webapp/app/shared/table/VusTable.tsx

```diff
@@ -232,7 +232,7 @@ const VusTable = ({
   return vusData ? (
     <div className={'justify-content-between align-items-center mt-4'}>
       <div className={'d-flex align-items-baseline'}>
-        <span style={{ fontSize: '2rem' }}>Variants of Unknown Significance (Investigated and data not found)</span>
+        <h5>Variants of Unknown Significance (Investigated and data not found)</h5>
```

Comment:
I want to keep the button on the same level of the header. Let's adjust fontsize instead